modules/gdscript/gdscript.cpp

```diff
@@ -629,6 +629,10 @@ void GDScript::_update_doc() {
 		}
 	}
 
+	for (KeyValue<StringName, Ref<GDScript>> &E : subclasses) {
+		E.value->_update_doc();
```

Comment:
@rune-scape I restored the `_update_doc()` call here [(and removed it in `gdscript_compiler.cpp`)](https://github.com/godotengine/godot/pull/67714/files#diff-43fc69f36899762bf59f1cd731dfc0be1ff0d2b648271e443944e4116c1c28a6L2578) because it crashed otherwise on @KoBeWi's large project.

![Capture d’écran du 2022-11-15 04-12-49](https://user-images.githubusercontent.com/270928/201879365-31a8cc5f-c4f3-44e6-bc85-edbe1c795071.png)

Reply:
this looks exactly like my regression bug fixed by #68717! this may fix it because the subclass types aren't compiled yet, so theres no invalid data yet. can you see if https://github.com/godotengine/godot/commit/95a85c9058e9c2a7eab3f5a272ffbe741a6750d1 fixes this?

Reply:
95a85c9 seems to fix the issue! Thanks.